src/main/java/uk/gov/hmcts/reform/pip/data/management/models/external/Channel.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.pip.data.management.models.external;
+
+public enum Channel {
```

Comment:
Don't believe this should be in data-management. Responsibility is more on sub / channel management

Reply:
sorted.